cluster/addons/etcd-empty-dir-cleanup/etcd-empty-dir-cleanup.yaml

```diff
@@ -24,4 +24,4 @@ spec:
   dnsPolicy: Default
   containers:
   - name: etcd-empty-dir-cleanup
-    image: k8s.gcr.io/etcd-empty-dir-cleanup:3.1.10.0
+    image: k8s.gcr.io/etcd-empty-dir-cleanup:3.1.11.0
```

Comment:
I don't see a 3.1.11.0 in gcr.io.  The etcd version of etcd-empty-dir-cleanup is only for the version of etcdctl copied into the container image which hasn't changed between 3.1.10 and 3.1.11.

Reply:
I see. In that case, do you want me to:
- keep the tag at 3.1.10.0, or
- change it to 3.1.11.0 (to sync with etcd version used in the makefile) and push a new image?

Reply:
Sigh. To be consistent, let's publish 3.1.11.0..

Reply:
We could also cheat and tag the 3.1.10.0 image with 3.1.11.0 as well :grin:

Reply:
I'd prefer the former, to avoid messing up the tag wrt the underlying etcd version actually used to build the image.

That said, I failed with the following error while running `make push`:

```
The push refers to a repository [staging-k8s.gcr.io/etcd-empty-dir-cleanup]
41ad6ade37a0: Pushing [==================================================>]  14.32MB/14.32MB
8b8608fe70b0: Pushing [==================================================>]  14.32MB/14.32MB
c5183829c43c: Layer already exists 
read tcp [2a00:79e0:2:11:cdf:6d7d:727b:913e]:37568->[2a00:1450:400c:c06::52]:443: use of closed network connection
```

@krzyzacy @BenTheElder Is this somehow related to that docker auth issue? Leads on how to fix it?

Reply:
can you link me to the make push command?

Reply:
cd cluster/images/etcd-empty-dir-cleanup
make push

Reply:
hmm do you have `gcloud beta auth` working on your machine now?

Reply:
I tried it. Getting:

```
ERROR: (gcloud.beta.auth.configure-docker) Error writing Docker configuration to disk: [Errno 2] No such file or directory: '/usr/local/google/home/shyamjvs/.docker/tmptYLal3'
```

Reply:
Ok.. I think I fixed it by creating that dir. Let me try pushing now.

Reply:
Still fails, but with:

```
unexpected EOF
```

Reply:
I think that's the gcloud docker -- push vs docker push issue again - have you upgraded your workstation yet?

Reply:
I finally managed to make it work by replacing `docker` with `gcloud docker` and using the right credentials allowing me to push.

@jpbetz - We're good to go here now.